tracer/test/Datadog.Trace.Tests/DatabaseMonitoring/DatabaseMonitoringPropagatorTests.cs

```diff
@@ -18,16 +20,22 @@ namespace Datadog.Trace.Tests.DatabaseMonitoring
 {
     public class DatabaseMonitoringPropagatorTests
     {
-        private readonly Tracer _tracer;
+        private readonly Tracer _v0Tracer;
+        private readonly Tracer _v1Tracer;
         private readonly Mock<IAgentWriter> _writerMock;
 
         public DatabaseMonitoringPropagatorTests(ITestOutputHelper output)
         {
-            var settings = new TracerSettings();
+            var v0Settings = new TracerSettings();
             _writerMock = new Mock<IAgentWriter>();
             var samplerMock = new Mock<ITraceSampler>();
 
-            _tracer = new Tracer(settings, _writerMock.Object, samplerMock.Object, scopeManager: null, statsd: null);
+            _v0Tracer = new Tracer(v0Settings, _writerMock.Object, samplerMock.Object, scopeManager: null, statsd: null);
+
+            var configSourceMock = new Mock<IConfigurationSource>();
+            configSourceMock.Setup(c => c.GetString(It.Is<string>(s => s.Equals(ConfigurationKeys.MetadataSchemaVersion)))).Returns(SchemaVersion.V1.ToString());
+            var v1Settings = new TracerSettings(configSourceMock.Object, new ConfigurationTelemetry());
```

Comment:
using Moq for `IConfigurationSource` is generally more hassle than it's worth 🙂 Instead, I think you can do something like this:

```suggestion
            var v1Settings = TracerSettings.Create(new(){ { ConfigurationKeys.MetadataSchemaVersion, SchemaVersion.V1.ToString() } });
```